src/main/java/com/example/blockchaininfo/services/Quartz/FindAndDisplayDataServiceQuartz.java

```diff
@@ -0,0 +1,77 @@
+package com.example.blockchaininfo.services.Quartz;
+
+import com.example.blockchaininfo.model.NetworkHashrate;
+import com.example.blockchaininfo.model.PoolHashrate;
+import com.example.blockchaininfo.repositories.NetworkHashrateRepository;
+import com.example.blockchaininfo.repositories.PoolHashrateRepository;
+import com.example.blockchaininfo.services.FindAndDisplayDataService;
+import com.example.blockchaininfo.services.GetDataService;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.quartz.*;
+import org.quartz.impl.StdSchedulerFactory;
+import org.springframework.context.ApplicationListener;
+import org.springframework.context.annotation.Profile;
+import org.springframework.context.event.ContextRefreshedEvent;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@RequiredArgsConstructor
+@Profile("quartz")
+@Slf4j
+@Service
+public class FindAndDisplayDataServiceQuartz implements ApplicationListener<ContextRefreshedEvent>, FindAndDisplayDataService {
+
+    private final NetworkHashrateRepository networkHashrateRepository;
+    private final PoolHashrateRepository poolHashrateRepository;
+    private final GetDataService getDataService;
+
+    @Override
+    public void onApplicationEvent(ContextRefreshedEvent contextRefreshedEvent){
```

Comment:
use formatter in IDE to automatically format code (i.e. spaces between ')' and '{')